python/lib/dependabot/python/file_updater/pip_compile_file_updater.rb

```diff
@@ -170,24 +170,6 @@ def run_pip_compile_command(command, allow_unsafe_shell_command: false)
             command,
             allow_unsafe_shell_command: allow_unsafe_shell_command
           )
-        rescue SharedHelpers::HelperSubprocessFailed => e
-          original_error ||= e
-          msg = e.message
-
-          relevant_error =
-            if error_suggests_bad_python_version?(msg) then original_error
-            else e
-            end
-
-          raise relevant_error unless error_suggests_bad_python_version?(msg)
-          raise relevant_error if user_specified_python_version
-          raise relevant_error if python_version == "2.7.18"
-
-          @python_version = "2.7.18"
-          retry
-        ensure
-          @python_version = nil
-          FileUtils.remove_entry(".python-version", true)
```

Comment:
This retry is ran when an update failed on python 3 and retries with python 2, this is now no longer required

Reply:
So nice getting rid of this!